models.py

```diff
@@ -3,38 +3,44 @@
 
 db = SqliteExtDatabase('elo.db', pragmas=(('foreign_keys', True),))
 
+needs_table_li = []
+def needs_table(x):
+    needs_table_li.append(x)
+    return x
+def create_tables():
+    for nt in needs_table_li:
+        nt.create_table()
+
+
 class BaseModel(Model):
     class Meta:
         database = db
 
-class Player():
-    """ Player is an *in memory only* object that is built up by replaying the match history """
 
-    def __init__(self):
-        self.wins = 0
-        self.losses = 0
-        self.rating = 1500
+@needs_table
```

Comment:
what is this

Reply:
Classes marked with `needs_table` will have a table generated for them by `create_tables`.